pyccl/core.py

```diff
@@ -207,6 +207,19 @@
     'bcm':         lib.bcm
 }
 
+# modified_gravity_types = {
+#    'GR':   lib.GR,
+#    'muSigma_MG':  lib.muSigma_MG
+# }
+"""dict: Types of modified gravity models.
```

Comment:
Good catch. I put this in a long time ago when we weren't sure what the scope of this PR was. I removed it, but it will be there in the commit history if someone wants to include it again with the implementation of other modified gravity options.